package.json

```diff
@@ -1,18 +1,24 @@
 {
   "name": "electron-vue-next",
   "version": "0.0.0",
+  "main": "dist/source/index.js",
   "private": true,
   "description": "",
   "scripts": {
-    "dev:docs": "node scripts/dev.docs.js",
-    "dev": "node scripts/dev.js",
-    "build": "node scripts/build.js",
-    "build:docs": "vitepress build docs",
-    "build:dir": "cross-env BUILD_TARGET=dir node scripts/build.js",
-    "build:lite": "cross-env BUILD_TARGET=lite node scripts/build.js",
-    "build:production": "cross-env BUILD_TARGET=production node scripts/build.js",
+    "build": "(concurrently --kill-others-on-fail \"npm:build:main\" \"npm:build:renderer\") && npm run build:app",
+    "build:main": "cross-env NODE_ENV=production rollup --config rollup.config.js",
+    "build:renderer": "cross-env NODE_ENV=production vite build ./src/renderer --outDir ../../dist/source/renderer",
+    "build:app": "electron-builder build",
```

Comment:
Will the electron-builder pack all the dependencies to the app.asar in this case? I think it will...